reporter-web-app/config/webpack.config.js

```diff
@@ -38,6 +38,8 @@ const shouldUseSourceMap = process.env.GENERATE_SOURCEMAP !== 'false';
 // makes for a smoother build process.
 const shouldInlineRuntimeChunk = process.env.INLINE_RUNTIME_CHUNK !== 'false';
 
+const isExtendingEslintConfig = process.env.EXTEND_ESLINT === 'true';
```

Comment:
commit-msg: In addition to updating the dev stack (dependencies) this commit applies a bunch of formatting fixes. Consider doing them in a separate commit.

Reply:
The format fixes come from upstream not from me..

Reply:
oh ok.